rdf_differ/entrypoints/api/handlers_helpers.py

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/python3
+
+# handlers_helpers.py
```

Comment:
we shall consider moving this functionality from the API into the service layer.